dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/loadbalance/ShortestResponseLoadBalanceTest.java

```diff
@@ -97,14 +97,13 @@ public void testSelectByResponse() throws NoSuchFieldException, IllegalAccessExc
             }
         }
         Map<Invoker<LoadBalanceBaseTest>, Integer> weightMap = weightInvokersSR.stream()
-                .collect(Collectors.toMap(Function.identity(), e -> Integer.valueOf(e.getUrl().getParameter("weight"))));
+            .collect(Collectors.toMap(Function.identity(), e -> Integer.valueOf(e.getUrl().getParameter("weight"))));
         Integer totalWeight = weightMap.values().stream().reduce(0, Integer::sum);
-        // max deviation 10%
+        // max deviation 12.5%
         int expectWeightValue = loop / totalWeight;
-        int maxDeviation = expectWeightValue / 10;
+        int maxDeviation = expectWeightValue / 8;
```

Comment:
As for test stability, it would be better to set maxDeviation to `expectWeightValue*2` to tolerant random seed problem.

Reply:
Good idea!